pkg/controller/middleware/template.go

```diff
@@ -40,9 +39,6 @@ func PopulateTemplateVariables(ctx context.Context, config *config.ServerConfig)
 			m["buildID"] = buildinfo.BuildID
 			m["buildTag"] = buildinfo.BuildTag
 
-			// Current page variables.
```

Comment:
This was being overwritten by the other middleware, but I lost about 30min trying to debug why `currentPath` wasn't a string. Removing for the future.